compile_assets/scss/admonition.scss

```diff
@@ -65,6 +65,8 @@ $admonitions: (
     --md-admonition-icon--#{nth($names, 1)}: 
       url("data:image/svg+xml;charset=utf-8,#{nth($props, 2)}");
   }
+  --md-details-icon:
+    url("data:image/svg+xml;charset=utf-8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><path d='M8.59 16.58 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42Z'/></svg>");
```

Comment:
I think this looks perfect! Thanks very much.

For reference, we have:

![image](https://user-images.githubusercontent.com/12255914/169036688-4ff5eae5-c639-47a9-a3f6-b61fb0f45de5.png)

and

![image](https://user-images.githubusercontent.com/12255914/169036728-d5dd69fd-ff37-43d0-aec8-94d6797e691f.png)